extensions/pooling/src/__tests__/unit/README.md

```diff
@@ -0,0 +1 @@
+# Unit tests
```

Comment:
How about some unit tests?

Reply:
The pooling service is a wrapper of `generic-pool`. I believe the extra functions have been covered by the acceptance tests.